tests/TestCase.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Orchestra\Testbench\TestCase as OrchestraTestCase;
+use Astrotomic\Translatable\TranslatableServiceProvider;
+
+abstract class TestCase extends OrchestraTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->loadMigrationsFrom([
+            '--database' => 'testing',
+            '--path' => realpath('tests/migrations')
+        ]);
+
+        $this->withFactories(realpath('tests/factories'));
+
+        app('config')->set('translatable.locales', ['el', 'en', 'fr', 'de', 'id', 'en-GB', 'en-US', 'de-DE', 'de-CH']);
```

Comment:
Should be done in `getEnvironmentSetUp()` - so far I know this method is still up2date and used to configure the environment.